### ISSUE TITLE
[Feature suggestion]: More intuitive inpainting

### ISSUE THREAD

**Issue by Woisek** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#232]
First, thanks for this great plugin. It's the second one I use for my PS works, and while it's quite powerful, it doesn't run as round as it could be.
My first and main plugin I use is this -> https://github.com/isekaidev/stable.art
What I absolutely LOVE is the fact, that is so much easier to select a region and inpaint away. Just use the Lasso and draw a selection, click generate, BAM, generation done.
With your plugin, I have to create a special layer, take a brush and paint, wait for some smart layer options and then, somethime later, it starts generating. And then, the results are seperated layers. One has to click on and off the layers to see the results. In stable.art, all generations are in one smart layer and only needs a click in the UI to look at them.

If you would implement this easy selecting approach (both masking and display) of stable.art, your plugin would be unbeatable.
Maybe you could consider in updateing it to such an easy approach. Would be much appreciated.
Thanks again for this powerful plugin.

**Comment by Renaldas111** [comment]
I must agree, inpainting in this plugin is the most unintuitive thing ever happened to me. And no, I still am not able to inpaint with this plugin, as this is beyond my understanding :)

**Comment by gobshyte** [comment]
Yeah I've just tried this plugin after using stable.art, as I wanted to do some outpainting, and yeah it would be excellent if this could be improved.
